test/channels/vehicles_channel_test.exs

```diff
@@ -30,6 +45,20 @@ defmodule SkateWeb.VehiclesChannelTest do
 
   describe "handle_info/2" do
     setup do
+      real_stop_times_on_trip_fn = Application.get_env(:realtime, :stop_times_on_trip_fn)
+
+      on_exit(fn ->
+        Application.put_env(:realtime, :stop_times_on_trip_fn, real_stop_times_on_trip_fn)
+      end)
+
+      Application.put_env(:realtime, :stop_times_on_trip_fn, fn _trip_id ->
+        [
+          %StopTime{stop_id: "6553", timepoint_id: "tp1"},
+          %StopTime{stop_id: "6554", timepoint_id: ""},
+          %StopTime{stop_id: "6555", timepoint_id: "tp2"}
+        ]
+      end)
```

Comment:
Why are these needed?

Reply:
Decoding vehicle data makes a call to get stop times on the trip: https://github.com/mbta/skate/blob/mss-vehicle-timepoint-data/lib/realtime/vehicle.ex#L93